platform/cervantes/koreader.sh

```diff
@@ -95,7 +101,7 @@ while [ "${RETURN_VALUE}" -ge "${RESTART_KOREADER}" ]; do
     find /mnt/public/dict -type f -exec mv -v \{\} /mnt/private/koreader/data/dict \; 2>/dev/null
 
     # run KOReader
-    ./reader.lua "${args}" >>crash.log 2>&1
+    ./reader.lua -d "${args}" >>debug.log 2>>crash.log
```

Comment:
That can be quite a few unnecessary writes on an SD card though (as a default), depending on what kind of internal memory it has.

Reply:
I have no idea about internal storage. Seems the same as other ntx boards (soldered? sdcard).

The partition where KOReader lives is ext3 mounted with noatime. Since it is a debian system there are quite a few logs already and they're not using tmpfs. 

But yeah, you're right "-d" probably shouldn't be the default setting.